fts-cron/renew_fts_proxy.sh.j2

```diff
@@ -0,0 +1,27 @@
+#! /bin/bash
+
+# We have to copy the certificates because we cannot change permissions on them as mounted secrets and voms-proxy is particular about permissions
+
+cp /opt/rucio/certs/usercert.pem /tmp/cert.pem
+cp /opt/rucio/keys/new_userkey.pem /tmp/key.pem
+chmod 400 /tmp/key.pem
+
+# Generate a proxy with the voms extension if requested
+voms-proxy-init2 --debug -rfc -valid 96:00 -cert /tmp/cert.pem -key /tmp/key.pem -out /tmp/x509up {% if RUCIO_FTS_VOMS is defined -%}-voms {{ RUCIO_FTS_VOMS }}{%- endif %} -rfc -timeout 5
+
+# Delegate the proxy to the requested servers
+{% if RUCIO_FTS_SERVERS is defined %}
+{% set ftses = RUCIO_FTS_SERVERS.split(',') %}
+{% for fts in ftses %}
+fts-rest-delegate  --key=/tmp/x509up --cert=/tmp/x509up -s {{ fts }}
```

Comment:
cf. one comment on slack:

suggestion, use the -f and -H parameter to delegate a longer-duration proxy (e.g., 96 hours) every hour. that way you still have a couple of hours left in case there's a glitch, eg. like so in the crontab:

`01 * * * * root (fts-rest-delegate -v -f -H 96 -s https://fts3-test.gridpp.rl.ac.uk:8446 --key /opt/rucio/etc/long.proxy --cert /opt/rucio/etc/long.proxy) >> /var/log/delegate_fts.log 2>&1`